subsystem/swerveIntakePivot.py

```diff
@@ -1,10 +1,87 @@
 import commands2
 import rev
-
-
-class SwerveIntakePivot(commands2.SubsystemBase):
+import math
```

Comment:
Should this be in this changeset or is swerveIntakePivot from another branch? If the latter, we can help get this cleaned up so this is ONLY for telemetry.